server/public/model/config.go

```diff
@@ -1052,6 +1053,10 @@ func (s *ExperimentalSettings) SetDefaults() {
 	if s.DelayChannelAutocomplete == nil {
 		s.DelayChannelAutocomplete = NewBool(false)
 	}
+
+	if s.EnableClientMetrics == nil {
+		s.EnableClientMetrics = NewBool(false)
```

Comment:
I was thinking of off-by-default because this is an experimental feature that requires additional configuration to set up Prometheus or the metrics plugin. I'll post in the channel to see what people are thinking for that

Reply:
Actually, Rudder "telemetry" is enabled by default, but the Prometheus "metrics" are off by default because of that. I'm going to move this setting under `MetricSettings` in a separate PR where it'll be on-by-default but behind a feature flag